core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -466,6 +467,20 @@ default void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath)
   void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, MountPOptions options)
       throws IOException, AlluxioException;
 
+  /**
+   * Same as {@link FileSystem#mount(AlluxioURI, AlluxioURI, MountPOptions)},
+   * and will return detailed mount execution process.
+   *
+   * @param alluxioPath an Alluxio path to mount the data to
+   * @param ufsPath a UFS path to mount the data from
+   * @param options options to associate with this operation
+   * @return details of the mount execution process
+   */
+  default List<String> mountWithDetail(AlluxioURI alluxioPath, AlluxioURI ufsPath,
+      MountPOptions options) throws IOException, AlluxioException {
+    throw new UnimplementedException("mountWithDetail");
+  }
```

Comment:
Can we make this a non-default method, and give the `void mount(...)` method a default implementation which forwards to this one with `options.detail` set to false?

Reply:
The `void mount(...)` method cannot return `List<String> message`, I don't want to change the signature of `mount` so I define a `mountWithDetail` method that can return a message. 
Using default is to reduce unnecessary implementations because not all implementations of FileSystem need to have `mountwithDeail` method